sentry_streams/src/fake_strategy.rs

```diff
@@ -1,33 +1,78 @@
 use super::*;
 use crate::routes::RoutedValue;
+
 use sentry_arroyo::processing::strategies::{
-    CommitRequest, ProcessingStrategy, StrategyError, SubmitError,
+    merge_commit_request, CommitRequest, InvalidMessage, MessageRejected, ProcessingStrategy,
+    StrategyError, SubmitError,
 };
-use sentry_arroyo::types::Message;
+use sentry_arroyo::types::{AnyMessage, BrokerMessage, InnerMessage, Message, Partition, Topic};
+use std::collections::HashMap;
 use std::sync::{Arc, Mutex};
 use std::time::Duration;
 
 pub struct FakeStrategy {
     pub submitted: Arc<Mutex<Vec<Py<PyAny>>>>,
+    pub reject_message: bool,
```

Comment:
These look to be separate changes from the `RustOperatorDelegate`-related work? What's the purpose of these changes being included in this PR?